Assets/Countly/Plugins/CountlySDK/Interfaces/IViewModule.cs

```diff
@@ -1,26 +1,126 @@
 using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-
-// interface for SDK users
-public interface IViewModule
-{
-    /// <summary>
-    /// Start tracking a view
-    /// </summary>
-    /// <param name="name">name of the view</param>
-    /// <returns></returns>
-    [Obsolete("RecordOpenViewAsync(string name, IDictionary<string, object> segmentation = null) is deprecated, this is going to be removed in the future.")]
-    Task RecordOpenViewAsync(string name, IDictionary<string, object> segmentation = null);
-
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+// interface for SDK users
+public interface IViewModule
+{
+    /// <summary>
+    /// Starts a view which would not close automatically
+    /// </summary>
+    /// <param name="viewName"></param>
+    /// <returns></returns>
+    string StartView(string viewName);
+
+    /// <summary>
+    /// Starts a view which would not close automatically
+    /// </summary>
+    /// <param name="viewName">name of the view</param>
+    /// <param name="viewSegmentation">segmentation that will be added to the view, set 'null' if none should be added</param>
+    /// <returns></returns>
+    string StartView(string viewName, Dictionary<string, object> viewSegmentation);
+
+    /// <summary>
+    /// Starts a view which would be closed automaticly
```

Comment:
automaticly? wow looks like someone created this pr and never went over ><
again what you wanted to say is probably akin to "Starts a view with the given name. Calling this again or starting any other view would close this one automatically. This ensures a 'one view at a time' flow."